web/src/components/organisms/Settings/MyIntegrationDetails/hooks.ts

```diff
@@ -0,0 +1,134 @@
+import { useCallback, useMemo } from "react";
+
+import { WebhookTrigger } from "@reearth-cms/components/molecules/MyIntegrations/types";
+import integrationHook from "@reearth-cms/components/organisms/Settings/MyIntegrations/hooks";
```

Comment:
I know you are only using integrations, but you are using the hooks file, that has multiple returned values, so it should be plural

`integrationHooks`

Reply:
Now it is `integrationHooks`.